airflow/providers/cncf/kubernetes/CHANGELOG.rst

```diff
@@ -19,6 +19,15 @@
 Changelog
 ---------
 
+
+1.2.1
+.....
+
+Bug Fixes
+~~~~~~~~~
+
+* ``Remove image_pull_policy "IfNotPresent" preset, will default to k8s "Always" if not set (#15388)``
```

Comment:
```suggestion
* ``Remove image_pull_policy "IfNotPresent" default as k8s already defaults to `Always` if :latest tag is specified, or `IfNotPresent` otherwise (#15388)``
* ``Remove service_account_name "default" default as k8s already defaults to `default` if not set (#15388)``
```

Feel free to adjust, but I think we should describe both removals.

Reply:
Yes, thank you.